src/Lumi/Components2/Banner.purs

```diff
@@ -0,0 +1,231 @@
+module Lumi.Components2.Banner
+  ( Banner
+  , BannerProps
+  , banner
+  , actionBanner
+  , primary
+  , active
+  , warning
+  , error
+  , module Styles.Banner
+  ) where
+
+import Prelude
+
+import Color (lighten)
+import Data.Foldable (fold)
+import Data.Maybe (Maybe(..), isJust)
+import Data.Monoid as Monoid
+import Data.Tuple.Nested ((/\))
+import Effect.Unsafe (unsafePerformEffect)
+import Lumi.Components (LumiComponent, PropsModifier, lumiComponent)
+import Lumi.Components.Icon as Icon
+import Lumi.Components.Spacing (Space(..))
+import Lumi.Components2.Box (box)
+import Lumi.Styles (color, css, int, nested, prop, str, style, style_, toCSS)
+import Lumi.Styles.Banner (banner) as S
+import Lumi.Styles.Border (_listSpaced, _listCompact) as Styles.Banner
+import Lumi.Styles.Box (FlexAlign(..), _align, _alignSelf, _column, _interactive, _justify, _row) as S
+import Lumi.Styles.Responsive (desktopQuery)
+import Lumi.Styles.Responsive (onDesktop) as S
+import Lumi.Styles.Theme (LumiTheme(..), useTheme)
+import React.Basic (JSX)
+import React.Basic.DOM as R
+import React.Basic.Emotion as E
+import React.Basic.DOM.Events (capture_)
+import React.Basic.Hooks (useState)
+import React.Basic.Hooks as React
+
+data Banner = Banner
+
+type BannerProps =
+  ( component :: Banner
+  , dismissable :: Boolean
+  , icon :: Maybe JSX
+  , title :: Maybe JSX
+  , content :: Array JSX
+  )
+
+banner :: LumiComponent BannerProps
+banner =
+  unsafePerformEffect do
+    lumiComponent "Banner" defaults \props -> React.do
+      theme <- useTheme
+      visible /\ setVisible <- useState true
+      pure
+        $ Monoid.guard visible
+        $ box
+        $ style props.css
+        $ S.banner
```

Comment:
The `props.css` overrides should come after `S.banner` now with the latest updates from #145.